src/containers/MealsPage/MealsPage.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import RecipeCard from '../../components/Cards/Recipe-Card/Recipe-Card';
+import DropDown from '../../components/DropDown/DropDown';
+//import breakfast1 from '../Cards/Recipe-Card/img/meals/breakfast/breakfast1.jpg';
+
+const mealList = [
```

Comment:
can you add the `meal.txt` file Mahmoud created that contains the JSON for the meals then fetch the meals from it?

you can use this as a reference 
https://github.com/Iraq-WBC-Capstones-2020/train-to-maintain-web/blob/master/src/containers/Workouts/Workouts.js